### DISCUSSION TITLE
Make zones clickable on Map panel

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11212] [Other feature requests]
On the Map panel, you can click on Device Trackers and Persons, but **not** Zones. This never made much sense to me. It feels weird to have some things you can click on and others you can't. 

![map](https://i.imgur.com/hH7k76G.png)

Clicking on the zone would open the more info dialog for zones, just like it does for device trackers / persons already. 

![zone](https://i.imgur.com/ujrrNf2.png)

And then if FR https://github.com/home-assistant/home-assistant-polymer/issues/4162 gets applied to more-info-zone as well, you'd have a useful shortcut into editing zones. Could be browsing the Map, see something is not right or needs to be tweaked, click on it, edit it, voila.

**Comment by bramkragten** [comment]
With the UI editor, this would be useful :-)
